source/partials/next_release/_release.md.erb

```diff
@@ -90,15 +66,14 @@
       <li><p><%= link_to('Strong Auth Plugin', 'https://github.com/danielsomerfield/go-strong-auth-plugin#go-strong-auth-plugin') %></p></li>
       <li><p><%= link_to('Guest Login', 'https://github.com/gocd-contrib/gocd_auth_plugin_guest_user') %></p></li>
     </ul>
-* <%= link_to_commit '8571bc13a55853ecff3f19e59f312b2e8b7a1e60', 'Removes garage controller and associated services', 3923 %>
-
 
 <h4>Note</h4>
-* With release 17.5.0 support for in-built password file and LDAP/AD authentication had been deprecated. In this release support for this has been removed in favour of the bundled <%=link_to('LDAP', "https://github.com/gocd/gocd-ldap-authentication-plugin")%>  and <%= link_to('Password File', "https://github.com/gocd/filebased-authentication-plugin") %> plugins respectively.
+* As of release 17.12, version 2 of the plugin info <%= link_to_api 'plugin-info' %> has been removed. Version 3 of the API is available, and users are encouraged to use it.
```

Comment:
Version 2 of the plugin info is still there. Not been removed yet. So, remove this note?

Reply:
Add a deprecation note for removing the plugin upload form in 18.2